subprojects/packagefiles/oatpp/meson.build

```diff
@@ -92,7 +86,15 @@ if is_msvc_or_similar
   )
 endif
 
-
+proj_flags = cpp.get_supported_arguments(
+  '-Wno-unused-parameter',
+  '-Wno-pessimizing-move',
+  '-Wno-unknown-warning-option',
```

Comment:
I'm okay with matching upstream here, but FWIW in Meson all dubious compiler options should be wrapped in `get_supported_arguments()` or similar, so this shouldn't be needed.